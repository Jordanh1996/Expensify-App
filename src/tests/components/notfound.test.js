import React from 'react';
import {shallow} from 'enzyme';
import Notfound from '../../components/notfound';

test('should render not found correctly', () => {
    const wrapper = shallow(<Notfound />)
    expect(wrapper).toMatchSnapshot()
})




