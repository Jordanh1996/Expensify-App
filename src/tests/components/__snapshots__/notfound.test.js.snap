// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/"
  >
    404
  </Link>
</div>
`;
